src/components/LoanCards/Buttons/LendAmountButton.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+	<lend-button
+		price="amountLeft"
+		:loan-id="loanId"
+		@add-to-basket="$emit('add-to-basket', $event)"
+	>
+		Lend ${{ amountLeft }} <span v-if="showNow">now</span>
+	</lend-button>
+</template>
+
+<script>
+import LendButton from './LendButton';
+
+export default {
+	components: {
+		LendButton,
+	},
+	props: {
+		loanId: {
+			type: Number,
+			default: null
+		},
+		loan: {
+			type: Object,
+			default: () => {}
+		},
+		showNow: {
+			type: Boolean,
+			default: false
+		}
+	},
+	computed: {
+		amountLeft() {
```

Comment:
This may end up being better set as a prop but it's fine to keep here for now. I suspect the the parent component  will need to calculate this to understand if we want to show this button or the lend increment button with the drop-down.

Reply:
ah yes... OK, i'll add a little `TODO` comment here to swap this for a prop if necessary in one of the later tix

Reply:
thinking about this now, wondering if it makes sense to turn this into a mixin if we have several cards calculating this

Reply:
or global js file

Reply:
We have a loanUtils file that could house a method to make this calculation.

Additionally, the LoanCardController already knows about this and passes it down to loan cards so you may already have it available in the loan cards you're working on.